plugins/chat/spec/requests/chat/api/channels_current_user_membership_controller_spec.rb

```diff
@@ -102,6 +102,74 @@
     end
   end
 
+  describe "#update" do
+    let!(:membership) do
+      Chat::UserChatChannelMembership.create!(
+        chat_channel_id: channel_1.id,
+        user_id: current_user.id,
+        following: true,
+      )
+    end
+
+    context "when pinning a channel" do
+      it "updates the pinned status" do
+        expect {
+          put "/chat/api/channels/#{channel_1.id}/memberships/me",
+              params: {
+                membership: {
+                  pinned: true,
+                },
+              }
+        }.to change { membership.reload.pinned }.from(false).to(true)
```

Comment:
Not sure we need that example, as we’re testing the same thing through the API response on line 126.